src/includes/performance/distributed-tracing/dotnet.mdx

```diff
@@ -0,0 +1,45 @@
+## Distributed Tracing
+
+Traces can bridge across multiple software services. Each span in a trace can be represented as a `sentry-trace` header, containing the trace id, span id, and sampling details. This `sentry-trace` header can be passed to downstream services so that they can create spans that are a continuation of the trace started in the originating service.
+
+To obtain a trace header from the span, use `ISpan.GetTraceHeader()` method, then pass it to the downstream service. If the communication happens over HTTP, it is recommended that you set the value to the `sentry-trace` HTTP header.
```

Comment:
turn out that this text is pretty similar to https://docs.sentry.io/platforms/java/performance/custom-instrumentation/#distributed-tracing (java.mdx), maybe we could actually move the generic text to the common file and only extract the important bits which is the method name.
wdyt @PeloWriter ?

Reply:
I think that would work.
- Add the content from line 3 "Traces can bridge..." to line 45 of `/common/performance/custom-instrumentation.mdx`
- Add .NET to the `<PlatformSection supported>` listed on line 42 of that same file

Reply:
I'm not sure if I understood it correctly, but I updated it. Can you please verify that I got it right?

Reply:
Verified that we now have that common content properly displaying for .NET and Java. Thank you.